site/cypress/e2e/Cypress-TAGrading/clearing_version_conflicts.spec.js

```diff
@@ -0,0 +1,41 @@
+describe('Test cases for checking the clear version conflicts button in the TA grading interface', () => {
+    it('Button should not appear if there are no version conflicts', () => {
+        cy.login('instructor');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'grade?who_id=LHmpGciilVzjcEJ&sort=id&direction=ASC']);
+        cy.get('[data-testid="grading-rubric-btn"]').click();
+        cy.get('[data-testid="change-graded-version"]').should('not.exist');
+        cy.get('[data-testid="version-warning"]').should('not.exist');
+    });
+
+    it('Button should appear if there are version conflicts', () => {
+        cy.login('instructor');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'grade?who_id=K8jI3q4qpdCc1jw&sort=id&direction=ASC&gradeable_version=1']);
+        cy.get('[data-testid="grading-rubric-btn"]').click();
+        cy.get('[data-testid="change-graded-version"]').should('exist');
+        cy.get('[data-testid="version-warning"]').should('exist');
+    });
+
+    it('Clicking the button should resolve those version conflicts', () => {
+        cy.login('instructor');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'grade?who_id=K8jI3q4qpdCc1jw&sort=id&direction=ASC&gradeable_version=1']);
```

Comment:
Would combining these tests into one make it slower at all? It seems like these could all be tested in the same test to reduce the amount of times the login is run, and since they are essentially three line tests, I think it would be good to combine all three into one slightly bigger test

Reply:
Combing them probably wouldn't negatively impact performance. I thought it was cleaner / easier to debug if they were in separate tests but if it doesn't matter that much I'm ok with combining them.

Reply:
Fixed in https://github.com/Submitty/Submitty/pull/10635/commits/1eb30dd58dd98393e45e9b61ffc1cdde0aeac89a